### ISSUE TITLE
Date filter doesn't work

### ISSUE THREAD

**Issue by xendk** [nordigen/nordigen-ruby#13]
**Describe the bug**
Even when providing `date_from` and `date_to` to `get_transactions` all transactions are returned.

**To Reproduce**
```ruby
# Filter transactions by specific date range
transactions = account.get_transactions(date_from: "2021-12-01", date_to: "2022-01-30")
```

**Expected behavior**
Only a subset of transactions returned.

**Additional context**
Looks like the Faraday call here https://github.com/nordigen/nordigen-ruby/blob/master/lib/nordigen-ruby.rb#L38 doesn't actually get the parameters.

**Comment by DanielJackson-Oslo** [comment]
Same issue here. I tried adding the params to the Faraday call by adding `conn.params = parameters`, but didn't manage to get it to work (haven't used Faraday before, so not sure if this was the right way to do it).

```ruby

def request(params = nil)
            # HTTP client request
            parameters = {}
            params&.each do |key, value|
                if value
                    parameters[key] = value
                end
            end

            @request ||= Faraday.new do |conn|
                conn.url_prefix = BASE_URL
                conn.headers = @@headers
                conn.request :json
                conn.response :json
                conn.params = parameters
            end
        end

```

Any work around? I guess fetching by HTTP instead of through the library?

**Comment by DanielJackson-Oslo** [comment]
Found a fix. See https://github.com/nordigen/nordigen-ruby/pull/15 :) Hope it works for you too!

**Comment by xendk** [comment]
Ah, but of course. I tried the same as you above, with equal success before giving up and filing the issue. Haven't tried it yet but it look much more right.

**Comment by victory-sokolov** [comment]
Fixed #15
